build.sh

```diff
@@ -82,6 +78,16 @@ do
 	j)
 	    JOBS=$OPTARG
 	    ;;
+	m)
+	    case $OPTARG in
+		debug | release)
+		    BUILD=$OPTARG
```

Comment:
Do we really need to specify which release we want?
Maybe we can add the -d option that will trigger a debug build, and leaving the default to release.

What do you think?

Reply:
I am proposing an obvious way to resolve the problem, i.e. specifying a build mode explicitly. This is a typical way that can be found in many build frameworks I have seen so far.
And what is more important, most of users will never know about this option because the release mode is set by default.
If you insist on your option we can change it later (as soon as I have time).

Reply:
You're supporting two alternatives (debug or release) with three combinations and with two being redundant (no option set and -m release). I'd rather prefer to have -d that toggles the debug build, just like gcc does if you need examples.